frontend/src/AlertConfiguration/AlertConfigurationComponents/AlertConfiguration.js

```diff
@@ -24,9 +23,7 @@ export default function AlertConfiguration() {
 
   const [configs, setConfigs] = useState([]);
 
-  fetch("/api/v1/configurations").then(response => {
-    console.log(response);
-  });
+  fetch("/api/v1/configurations").then(response => response.json()).then(res => setConfigs(res));
```

Comment:
just wondering when these fetch calls actually get called since they're async functions and `AlertConfiguration` is a pure functional component, from my understanding